components/cop_codegen/src/rpn_function.rs

```diff
@@ -284,24 +284,22 @@ mod tests {
     #[test]
     fn test_no_generic_generate_fn_trait() {
         let gen = no_generic_fn();
-        let expected: TokenStream = r#"
+        let expected: TokenStream = quote! {
```

Comment:
The explicit `: TokenStream` should no longer be needed. You may keep it if you think this is clearer.

Reply:
Yes, this also makes me a little hesitant, but I keep it make others can understand easily.